addons/littlebird_hud/fn_hud.sqf

```diff
@@ -1,15 +1,22 @@
 if !(cnto_littlebird_hud_enable) exitWith {};
 params ["_vehicle"];
-
+private _isLittlebird = _vehicle isKindOf "RHS_MELB_AH6M";
 // AH-6 is zeroed to eye level of player, not vehicle centre. AH-9 is zeroed to centre of aircraft model. Therefore offset Z level of reticle if vehicle is AH-6
-private _zOffset = [0, 25] select (_vehicle isKindOf "RHS_MELB_AH6M");
+private _zOffset = [0, 25] select _isLittlebird;
 private _size = 1 / (getResolution select 5);
 
+// Remove grease dot on vehicle entry
+if (_isLittlebird) then {
+    _vehicle animateSource ["Addcrosshair", 0, true];
+} else {
+    _vehicle setObjectTextureGlobal [1, "\A3\Weapons_F\Data\clear_empty.paa"];
```

Comment:
Maybe the non-global variant would suffice.

Also, you might want to save `getObjectTextures#1` (or so) and just restore it later, rather than hardcoding `heli_light_01_dot_ca.paa` below.

Reply:
Even though it's tiny, IMO things should be as consistent between all players as possible.

Reply:
Actually no I'm a dumbass passengers will overwrite the pilot this way, should do it locally.

Reply:
Although animateSource is always global, hm.